diff --git a/backend/migrations/2023-09-07-141528_migrate-payout-settings/up.sql b/backend/migrations/2023-09-07-141528_migrate-payout-settings/up.sql
index a3dc5719f2..aa5914981f 100644
--- a/backend/migrations/2023-09-07-141528_migrate-payout-settings/up.sql
+++ b/backend/migrations/2023-09-07-141528_migrate-payout-settings/up.sql
@@ -78,6 +78,10 @@ WHERE
payout_settings IS NOT NULL;


--- finally, drop payout_settings column
+-- finally, drop payout_settings columns
+ALTER TABLE user_payout_info
+DROP COLUMN are_payout_settings_valid;
+
+
ALTER TABLE user_payout_info
DROP COLUMN payout_settings;
\ No newline at end of file